docs/source/teams/installation.rst

```diff
@@ -447,116 +447,67 @@ alias:
     # For example
     filepath = "minio://test-bucket/image.jpg"
 
-Cloud Credentials Settings Page
-_______________________________
+.. _teams-cloud-storage-page:
 
-Credentials can also be configured via the Settings page of Teams. Note that
-only users with the Administrator role have access to this page.
+Cloud storage page
+__________________
 
-From the Settings page, navigate to ``Cloud storage``
+Admins can also configure cloud credentials via the Settings > Cloud storage
+page.
 
-This page lists all of the credentials that have been added to the Teams database
-via the UI.
+Credentials configured via this page are stored (encrypted) in the Teams
+database, rather than needing to be configured through environment variables in
+your Teams deployment.
 
 .. note::
 
-    Credentials provided via environment variables to local machines
-    or directly to deployed containers will not be displayed in this menu.
+    Any credentials configured via environment variables in your deployment
+    will not be displayed in this page.
+
+To upload a new credential, click the ``Add credential`` button:
 
 .. image:: /images/teams/cloud_creds_add_btn.png
     :alt: cloud-creds-add-credentials-button
     :align: center
 
-To provide a new credential to users, click the ``Add credential`` button. This will
-bring up a modal that gives you options to add credentials for the four providers
-described above (GCP, AWS, Azure, and MinIO).
+This will open a modal that you can use to add a credential for any of the
+available providers:
 
 .. image:: /images/teams/cloud_creds_modal_blank.png
     :alt: blank-cloud-creds-modal
     :align: center
-    :width: 50%
-
-
-For each of the providers, the appropriate options are listed for fields to populate.
-
-+--------------+--------------------------+
```

Comment:
@brimoor did you intend to remove this?

Reply:
Yes it's not really helpful; it was partially explaining what you see when using the UI pictured above, which is already self-explanatory IMO